src/formats/cif.cpp

```diff
@@ -0,0 +1,205 @@
+//
+// Created by danny305 on May 3rd, 2021.
+//
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include <stdexcept>
+#include <fstream>
+#include <filesystem>
+
+#include <gemmi/cif.hpp>
+#include <gemmi/mmcif.hpp>
+
+#define GEMMI_WRITE_IMPLEMENTATION
```

Comment:
This is not available in the old gemmi library (0.3.6). Update requirements in README to some current version like 0.4.7.

Reply:
Sure.